src/ui/dashboard.js

```diff
@@ -50,6 +50,30 @@ export class DashboardScene extends Phaser.Scene {
             this.currentScene.scene.start(CONSTANTS.SCENES.MAIN_MENU);
         });
 
+        // XP Counter
+        this.xpCounterOn = true;
+        this.xpCounterOnButton = this.add
```

Comment:
Add these 2 to the class member variables at the top and remove `this.xpCounterOn = true` from here